docs/topics/http/file-uploads.txt

```diff
@@ -126,6 +126,19 @@ model::
             form = UploadFileForm()
         return render(request, 'upload.html', {'form': form})
 
+If you are constructing an object manually outside of a request, you can assign
+the :class:`~django.db.models.FileField` to a :class:`~django.core.files.File`
+like object on the model instance::
```

Comment:
As far as I'm aware `FileField` and `File` should be swapped in this sentence
```suggestion
a :class:`~django.core.files.File` like object to the
:class:`~django.db.models.FileField`::
```

Reply:
Makes sense. Fixed!